index.html

```diff
@@ -16,20 +16,29 @@
     <script src="https://kit.fontawesome.com/6d267a36ba.js" crossorigin="anonymous"></script>
     <title>Yegon's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="portfolio.html">
 </head>
 <body>
+    <!-- Main Portfolio -->
     <section class="main">
-        <div>
             <nav>
-            <div class="h-menu">
-                <a href="">Welcome</a>
-                <div>
-                <div class="bar"></div>
-                <div class="bar"></div>
-                <div class="bar"></div>
-                </div>   
-             </div>
-            </nav>
+                <div class="main-menu">
+                <div class="h-menu">
+                    <a href="" class="greet">Welcome</a>
+                    <i class="fa-solid fa-bars"></i>
+                </div>
+                <div class="hidden-menu">
+                    <div class="hid-menu">
+                        <div class="close-btn">&times;</div>
+                        <ul class="hidden-cont">
+                        <li><a href="index.html">Portfolio</a></li>
+                        <li><a href="./html/about.html" class="h-abt">About</a></li>
+                        <li><a href="./html/contact.html" class="h-contact">Contact</a></li>
+                        </ul>
+                    </div>
```

Comment:
- [ ] [ **OPTIONAL** ] Good job adding the mobile menu to your portfolio :clap: , however I noticed that when the menu is active you can still scroll, through the page, It could be better if change that for an excellent user experience :+1: 

[Screencast from 03-07-2023 10_56_32 AM (1).webm](https://user-images.githubusercontent.com/80458199/223373060-42b5588c-39bb-48cd-bf41-3347c100a7ee.webm)